### DISCUSSION TITLE
next steps for new home page

### DISCUSSION THREAD

**Discussion by laurelsilva** [eggheadio/egghead-next#936] [Pitches]
https://github.com/eggheadio/egghead-next/pull/934

The next steps are to personalize the home page to the 3 types of users:

- Logged-out users
- Logged-in users with no subscriptions
- Logged in users with subscriptions

Logged-out and logged-in users should see the Checkout Widget at the bottom of the page. We must communicate to someone that doesn't know egghead that this is a paid product. It should be apparent from skimming the page that this is a paid product. 

Logged-in users should not see CTA's to create an account. 

Move the existing home page to `/learn`. 

Logged-in users with subscriptions should still be able to see the home page. This is not an issue, and it's a good reminder as to why they have an egghead membership.  

Icons should link to their respective topic page for logged-in users, whether they have a subscription or not. If they are logged-out users, we want to encourage them to read the long-form landing page. 

<img width="1508" alt="Untitled" src="https://user-images.githubusercontent.com/57044804/139455639-bdd14f93-a0ed-40d3-9fb9-b80c3add4910.png">

In the navbar, Browse all topics should point to the `/learn` route. 

![Untitled (1)](https://user-images.githubusercontent.com/57044804/139455672-209a0898-aa5a-4d49-a9bb-9c1ffdf5b443.png)

The home page should be personalized with the preferred topic for logged-in users, whether they have a subscription or not. **This might a separate PR. Everything above is needed to get this shipped.** 

![Untitled (2)](https://user-images.githubusercontent.com/57044804/139455693-84dd4b20-341c-4d86-b5b5-99f49a5bad52.png)

What's missing here?

**Comment by joelhooks** [comment]
the literal “all topics” page is still important and that the /learn route is interesting but buries MOST of the catalog so what the browse drop down exposes is an interesting puzzle


these choices could be quantified by understanding how people actually use the site


for now it probably makes sense to not be concerned with “personalization” and focus strictly on how the page is presented to logged in users so it can be published which is a lot less than described here?

**Reply by laurelsilva** [reply]
I agree. personalization is a different pitch/PR. 

I wonder where this page fits in https://egghead.io/courses. It contains all the necessary information if you want to find out what's new. It doesn't bury the catalog.

**Comment by joelhooks** [comment]
it’s definitely another way to look at the content that should be considered. Both that page and topics are very “raw” but have potential
